### ISSUE TITLE
Since Updated to Woocommerce 9.3.1 Theres been lots of OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to oauth.dnapayments.com:443 Errors

### ISSUE THREAD

**Issue by Raydelta** [DNA-Payments/woocommerce-dnapayments-gateway#3]
Since updated to Woocommerce 9.3.1 15/09/24 I have seen 39 instances of this on the logs today (16th Sept). We had 1000+ orders this day.

One customer said it happened after they paid and payment authorised, they still got their order confirmation email and the order processed correctly, however;

The customer is served - 500 - Internal Server Error which is the grey page that states "There has been a critical error on this website".

My Curl Version: 7.64.0, OpenSSL/1.1.1n
PHP Version: 7.4.33
DNAPayments Plugin Version: 2.3.0 (I cannot update to the latest version that you have here on github as that is requiring php 8.1, as per the other issue I raised).

DNA was using Iframe Lightbox setting. I am on AWS server which is large, CPU usage even at busiest times is sub 12%.

My Cloudflare is set to full ssl/tls with no issues, using cloudflares recommended settings.
Logs attached.

[fatal-errors-2024-09-16.log](https://github.com/user-attachments/files/17021136/fatal-errors-2024-09-16.log)

**Comment by Raydelta** [comment]
Note I am using standard Woocommerce shortcodes for Cart and Checkout.

**Comment by Raydelta** [comment]
I noticed I had SSL on @ cloudflare only with "TLS/SSL - Full" so SSL cert @ the edge only. I just installed a 15 year cert on the apache server as there wasn't one at origin, just edge, & moved edge settings for TLS/SSL on cloudflare to "Full (Strict)". As I noticed afew hundred requests not on TLS 1.2 or 1.3, vs 300k+ on TLS 1.3. I will keep checking logs today?

As the error log seems to indicate that specifically when attempting to connect to the URL oauth.dnapayments.com:443 over HTTPS it threw the error.

As it was only 39 out of 1000+ orders, I am assuming, that these fell into the non secure traffic (between edge and origin). But I am just assuming, not sure. Will keep checking logs today
